server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -225,10 +227,10 @@ object CromwellEntryPoint extends GracefulStopSupport {
   def validateRunArguments(args: CommandLineArguments): WorkflowSourceFilesCollection = {
 
     val sourceFileCollection = (args.validateSubmission(EntryPointLogger), writeableMetadataPath(args.metadataOutput)) mapN {
-      case (ValidSubmission(w, u, r, i, o, l, Some(z)), _) =>
+      case (ValidSubmission(s, u, r, i, o, l, Some(z)), _) =>
```

Comment:
Should we really rename a single variable? And if so, why not call it `ws` for better clarity of this?

Reply:
This variable confused me every time when I saw it until I renamed it.
`w` looks like it is a sufficient stand-alone variable about workflow, although it is not. It stands specifically for `source`, not for just `workflow`.
If I rename it to a `ws`, then it is better to rename other variables like `u` to `wu`, `r` to `wr` etc. But it turns out that the same parameters have different names in similar classes. For example, the same parameter can be called `options` and `workflowOption`. Without changing a lot of places it will just make everything worse while changing it shadow the purpose of the PR.

Reply:
Oh, now I saw that all these variables are related to workflow and do not have `w` prefix. Therefore, sure this is a good idea to rename it to `s`.